doc/SPONSORS.in

```diff
@@ -29,6 +24,17 @@ Treehouse Networks, NZ - http://treenet.co.nz/
 	toward Squid-3+ development and maintenance for their customer
 	gateways and CDN.
 
+@Squid-6:
+DigitalOcean - https://www.digitalocean.com/
+
+	DigitalOcean has donated droplets from their cloud infrastructure
+	to host most of Squid Project's continuous integration farm.
```

Comment:
AFAICT, DigitalOcean donations have not stopped and are used for master/v7 CI testing (among other things). Thus, their blob should be in the top/Squid-7 group (because we do not duplicate blobs and use the highest relevant Squid version to determine each sponsor blob placement).

Reply:
done in [c28c60b](https://github.com/squid-cache/squid/pull/1321/commits/c28c60b226fd93c012fb9985005d2b2d15ea4d4e)